File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.SearchService.Models/get-instructors_item.ts


import { IGetInstructors_Item } from './iget-instructors_item';

export class GetInstructors_Item implements IGetInstructors_Item {
  
	public PersonID: number = 0;
	public FirstMiddleNames: string = "";
	public LastNames: string = "";
	public DOB?: Date;
	public CountryID: number = 0;
	public CountryName: string = "";
	public CountryFullName: string = "";
	public UnitName: string = "";
	public UnitNameEnglish: string = "";
	public UnitMainAgencyID?: number;
	public Rating?: number;
  
}

File: GTTS/INL.SearchService/Models/GetInstructors_Item.cs
using System;

namespace INL.SearchService.Models
{
    public class GetInstructors_Item : IGetInstructors_Item
    {
        public long PersonID { get; set; }
        public string FirstMiddleNames { get; set; }
        public string LastNames { get; set; }
        
        
        
        public DateTime? DOB { get; set; }
        public int CountryID { get; set; }
        public string CountryName { get; set; }
        public string CountryFullName { get; set; }
        public string UnitName { get; set; }
        public string UnitNameEnglish { get; set; }
        public long? UnitMainAgencyID { get; set; }
        public decimal? Rating { get; set; }
    }
}
